tests/testthat/test-templates.R

```diff
@@ -74,9 +74,11 @@ test_that("find templates in local pkgdown first", {
 
 test_that("BS5 templates have main + aside", {
   names <- dir(path_pkgdown("bs5", "templates"), pattern = "content-")
```

Comment:
```suggestion
  names <- dir(path_pkgdown("BS5", "templates"), pattern = "content-")
```

Reply:
Oooooooooooooooooooooh

Reply:
You have no idea how much time this has saved me 🤣 also a good reason why using fs functions are safer than base.

Reply:
so base R functions are not case-sensitive?!

Reply:
No, it's just that `dir()` returns `character()` when the directory doesn't exist; `dir_ls()` errors.